cluster/gce/trusty/node.yaml

```diff
@@ -139,6 +139,8 @@ script
 		--configure-cbr0=true \
 		--cgroup-root=/ \
 		--system-container=/system \
+                --runtime-container=/docker-daemon \
+                --resource-container=/kubelet \
```

Comment:
If I understood correctly, the tab is required by whatever version of cloud_init version they are using. But will leave @andyzheng0831 for last answer.